app/controllers/eredienst-mandatenbeheer/new.js

```diff
@@ -48,11 +50,16 @@ export default class EredienstMandatenbeheerNewController extends Controller {
     event.preventDefault();
 
     let { worshipMandatee } = this.model;
-    yield worshipMandatee.save();
+    if (this.shouldSelectMandaat) {
```

Comment:
I would use an async util for this instead. That way the state can simply be derived from the mandatee record so we don't need to keep a custom flag here.

```suggestion
    if (shouldSelectMandaat(worshipMandatee)) {
```

The util could then simply check if the mandate relationship is set.